packages/api-client/src/collections.ts

```diff
@@ -63,7 +63,7 @@ export const updateCollection = async (params: {
       httpMethod: 'PUT',
       resource: '/{proxy+}',
       headers: { 'Content-Type': 'application/json' },
-      path: `/collections/${collection.name}/${encodeURIComponent(collection.version)}`,
+      path: `/collections/${encodeURIComponent(collection.name)}/${encodeURIComponent(collection.version)}`,
```

Comment:
Nit: Just for the sake of consistency could we do a helper function here like we have in `encodeGranulesURIComponent` in api-client/src/granules.ts?

Reply:
Also could you remind me where the decoding happens:? The api-client receives the request, encodes the params, and then passes it off to the appropriate API endpoint. Does the... router... handle decoding the URI encoded params? It looks like it does happen, just having a total lapse as to... where 😆